internal/metrics/metrics.go

```diff
@@ -36,10 +38,17 @@ var (
 			Help: "Emits one when provider secret is found to be absent, or zero when expected secrets exist",
 		},
 		[]string{mzRecordNameLabel, mzRecordNamespaceLabel, mzSecretNameLabel})
+	RecordReady = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "dns_provider_record_ready",
```

Comment:
There was a question over using "_ready" or "_info". I think ready is fine, as it is what the value of the metric is indicating. From my understanding of the last few days, if the metric has the "_info" suffix the value of the metric would not be expected to change, and should always be set to 1.

Reply:
This is the way I understood Dave as well. 
It just seems... wasteful to me to disregard a metric value  - this is a piece of information that could be used. 
That said, if we did not have a concept of record being ready, I'm not sure I could justify anything else as a "value" of the metric :thinking: